src/python/tools/Report_InvalidUrls_FindHttpURLs.csv

```diff
@@ -0,0 +1,97 @@
+url,relativeFilepath
```

Comment:
>url,relativeFilepat [](start = 0, length = 19)

double check manually that the urls here are really non valid. And update your script to handle valid cases